### PR TITLE
[MU4] fix #7675 - simplify page inspector for score/page

### LINKED ISSUE #7675: [MU4 Issue] Score breaks if we select note and double click on empty score space
**Describe the bug**
Score zoom out and notes zoom in  (video attached)

**To Reproduce**

1. Create a score 
2. Add notes
3. Click on a note to select it
4. Double click left mouse button on empty space on a score

**Expected behavior**
Note deselected

**Screenshots**

https://user-images.githubusercontent.com/71372310/110196280-92ca0d80-7e54-11eb-8d27-6d8e9c7f52e6.mp4

**Desktop (please complete the following information):**

**Additional context**
Bug Reproduces with this score
https://github.com/musescore/MuseScore/blob/master/vtest/scores/lyrics-8.mscx

### PR DESCRIPTION
**Pull request by wizofaus**
Resolves: https://github.com/musescore/MuseScore/issues/7675

As per @Tantacrul, the page size dropdown on this inspector that was causing the issue isn't needed, nor are any of the other settings.

- [ x ] I signed [CLA](https://musescore.org/en/cla)
- [ x ] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [ x ] I made sure the code compiles on my machine
- [ x ] I made sure there are no unnecessary changes in the code
- [ x ] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [ x ] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [ x ] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [ ] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

File: src/inspector/view/qml/MuseScore/Inspector/score/ScoreAppearanceInspectorView.qml [hunk 1]
```diff
@@ -187,10 +49,10 @@ InspectorSectionView {
                 width: parent.width
 
                 navigation.panel: root.navigationPanel
-                navigation.name: "More page settings"
+                navigation.name: "Page settings"
                 navigation.row: root.navigationRow(6)
 
-                text: qsTrc("inspector", "More page settings")
+                text: qsTrc("inspector", "Page settings")
```
Comment (wizofaus) [inline src/inspector/view/qml/MuseScore/Inspector/score/ScoreAppearanceInspectorView.qml:55]: Note that there is already a translation for "Page settings", but not "More page settings"

**Comment by igorkorsukov** [comment]
@wizofaus Would you fix the conflicts?

**Comment by wizofaus** [comment]
Yep, will do

File: src/inspector/view/qml/MuseScore/Inspector/score/ScoreAppearanceInspectorView.qml (hunk 1 above)
Comment (Eism) [reply src/inspector/view/qml/MuseScore/Inspector/score/ScoreAppearanceInspectorView.qml:55]: @Tantacrul what do you think of this change?

Comment (Tantacrul) [reply src/inspector/view/qml/MuseScore/Inspector/score/ScoreAppearanceInspectorView.qml:55]: Does this relate to the button that triggers the more settings dialog? If so, I don't exactly know why we'd add the word 'more'. @wizofaus Pretty much all page settings are in that dialog, no?

Perhaps I misunderstanding the string change

Comment (wizofaus) [reply src/inspector/view/qml/MuseScore/Inspector/score/ScoreAppearanceInspectorView.qml:55]: You agreed none of the page-specific settings that were there actually made much sense to be directly on the inspector, so I removed them all, hence that button is now for *all* page settings (not just ones in addition to whatever was there before).
Also note there's no translation for "More page settings", but there is for "Page settings".

Comment (Tantacrul) [reply src/inspector/view/qml/MuseScore/Inspector/score/ScoreAppearanceInspectorView.qml:55]: I did agree to all those things. I think I misunderstood the string change. I thought it was the other way around. Apologies. Rushing

**Pull request merged**